docs/content/querying/lookups.md

```diff
@@ -323,6 +323,8 @@ It is possible to save the configuration across restarts such that a node will n
 |Property|Description|Default|
 |--------|-----------|-------|
 |`druid.lookup.snapshotWorkingDir`| Working path used to store snapshot of current lookup configuration, leaving this property null will disable snapshot/bootstrap utility|null|
+|`druid.lookup.numLookupLoadingThreads`| Number of threads for loading the lookups in parallel on startup. This thread pool is destroyed once startup is done. It is not kept during the lifetime of the JVM|Available Processors / 2|
+|`druid.lookup.enableLookupSyncOnStartup`|Enable the lookup synchronization process with coordinator on startup. The queryable nodes will fetch and load the lookups from the coordinator instead of waiting for the coordinator to load the lookups for them. This option is recommended only in cases where the queryable nodes have lookups configured|false|
 
```

Comment:
@leventov could you review this doc update?

Reply:
What does it mean "the queryable nodes have lookups configured"?

Reply:
I have modified it as follows:

Enable the lookup synchronization process with coordinator on startup. The queryable nodes will fetch and load the lookups from the coordinator instead of waiting for the coordinator to load the lookups for them. This option is recommended only in cases where the user has configured lookups and in which case lookups would be immediately available for querying once the node startup is complete.
Please review.

Reply:
It doesn't answer what "the user has configured lookups" mean. How that state is different from "user has not configured lookups"?

Reply:
What I meant was it is pointless to enable this for cases the user has not posted lookups, as it would trigger an unnecessary coordinator request. I can skip/rephrase that statement, if you feel so.

Reply:
Ok, I see now. In this case it seems to me that it's better to remove this option, and make the sync always on. Because minimizing the number of options is an absolute value, and making some pointless queries doesn't seem very harmful

Reply:
If the sync is always turned on, users will see an unexpected delay in node startup time because it has to load all the lookups that the coordinator has provided. But if the user enables this option explicitly, they know they want lookups to be loaded upfront with a potential increase in startup. I have rephrased it as follows:

Enable the lookup synchronization process with coordinator on startup. The queryable nodes will fetch and load the lookups from the coordinator instead of waiting for the coordinator to load the lookups for them. This option is recommended only in cases where the user has configured lookups and requires the lookups to be available as soon as the node startup is complete. This can slightly increase the startup time of the node.

Reply:
Why one would not want to make initialization slower, getting rid of the race condition in return?

Reply:
I think it might be alright to turn it on by default considering a wasteful request in the event of user not using lookups would not be so bad.
However, I would like to keep the configuration to have option of restoring older behavior in case any problems are discovered due to loading all lookups at startup in any of the user clusters. @drcrallen do you have any other opinion ?

Reply:
I have enabled it by default. @leventov But do you agree that we should keep the option available so that the user has an option to turn it off in the event of identifying any issues that could pop up due to lookup loading at startup?